src/controllers/controller.bubble.js

```diff
@@ -10,7 +10,8 @@ var resolve = helpers.options.resolve;
 
 defaults._set('bubble', {
 	hover: {
-		mode: 'single'
+		mode: 'nearest',
```

Comment:
I think `{mode: 'nearest', intersect: true}` is the default, so you shouldn't need this. Same elsewhere as well

https://www.chartjs.org/docs/latest/general/interactions/?h=intersect

Reply:
done (not in tests / samples though)

Reply:
I don't know if you have to do an exhaustive search of all tests / samples, but it might be nice to update the tests that are already in this PR (or at least some of them to make sure that we're testing the defaults)

Reply:
I think we have plenty of tests for the remaining modes. Can't be arsed to verify if we are testing for defaults or not (for each dataset) 😫

Reply:
Oh yeah, I definitely wasn't suggesting to update each dataset type. I was only thinking that there's no reason to specify `{mode: 'nearest', intersect: true}` in so many tests in core.tooltip.tests.js since it's already the default

Reply:
True, but tests should not change, when defaults change (unless testing the defaults) - so being explicit there should be good.

Reply:
Agree on not changing it all at once. This gives us some more peace of mind